thirdparty/spdlog-20170710/include/spdlog/details/os.h

```diff
@@ -459,9 +459,9 @@ inline bool in_terminal(FILE* file)
 {
 
 #ifdef _WIN32
-    return _isatty(_fileno(file)) ? true : false;
```

Comment:
While we do have some specializations changing a third party library for readability and not functionality might not be the most ideal if we have to compare across versions.